src/app/cases/page.tsx

```diff
@@ -1,58 +1,73 @@
 'use client';
 
 import { useEffect, useState } from 'react';
-import { getAllCases } from '../../api/supabase/queries/cases';
+import styled from 'styled-components';
+import { UUID } from 'crypto';
 import { CaseListing } from '../../types/schemaTypes';
+import { getNCases } from '../../api/supabase/queries/cases';
+import ListingCard from '../../components/ListingCard';
+
+// styled components
+const PageContainer = styled.div`
```

Comment:
thoughts on styled components in a separate file? @pragyakallanagoudar

Reply:
agreed! @jinkang-0 can you move the components into a separate file called `styles.tsx` under `src/app/cases` and then import from there

Reply:
for the page only or should i do this for all components?

Reply:
@davidqing6432 @pragyakallanagoudar should i apply this change for this page only or for all components?

Reply:
both! for components, you want to separate into, say,
`src/components/CardTag/CardTag.tsx`
and
`src/components/CardTag/styles.tsx`

Reply:
@pragyakallanagoudar hmm I get the change can declutter most files, but for small files like `CardTag.tsx`, is it really necessary to split them into two separate files?

Reply:
GREAT pushback!! last semester [Cultural Survival](https://github.com/calblueprint/cultural-survival-mobile) used the approach you suggested, i.e. separating for pages but not components.

@davidqing6432 thoughts on this?